src/items/generics.md

```diff
@@ -83,6 +83,56 @@ where
 }
 ```
 
+## Free variables and concrete items and types
+
+The *free variables* of an item or type are the type and lifetime variables
+that refer to a generically declared parameter. A *concrete item* or *concrete
+type* is one that does not have any free variables. See [Wikipeida][concrete
+wikipedia] for more.
```

Comment:
Some general notes about this paragraph that I think are not well captured:
+ Bounds/Constraints can also have free variables, e.g. `u8: 'a` has `'a` as a free variable.
+ A lifetime can be concrete: `'static`.